javascript/javascript1/week4/class_exercises/sentiment_analyzer.js

```diff
@@ -0,0 +1,37 @@
+const positivePossibilities = [
+    "mega", "awesome", "super", "excited", "positive", "optimistic", "good", "excellent", "energetic",
+];
+
+const negativePossibilities = [
+    "angry", "negative", "pessimistic", "sad", "frustrated", "bad", "down", "sick", "tired",
+];
+
+
+function wordMatcher (wordsPossibilities, sentence) {
+    const matchesArray = [];
+    for (word of wordsPossibilities) {
+        if (sentence.includes(word)) {
```

Comment:
⚠️"Includes" is a powerful tool, but it has a drawback:

What would happen if instead of "_sad_" I wrote "_saddleback_"? The program would count the word as a feeling because it "includes" "sad" as a substring.

💡 One way to avoid this issue could be to split the sentence into individual words and then check each word for equality.